### PR TITLE
Make our CUDA code be compatible with the latest VS2019

### PR DESCRIPTION
**Pull request by snnn**
**Description**: 

Make our CUDA code be compatible with the latest VS2019

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

**Review by pranavsharma — APPROVED** [review]
Thanks for unblocking!

File: onnxruntime/core/providers/cuda/tensor/resize_impl.cu [hunk 1]
```diff
@@ -10,29 +10,29 @@ using onnxruntime::UpsampleMode;
...
   if (x_original == static_cast<int>(x_original) + 0.5f) {
-    return static_cast<int>(floor(x_original));
+    return static_cast<int>(_Floor(x_original));
   }
-  return static_cast<int>(round(x_original));
+  return static_cast<int>(_Round(x_original));
 }
 
 __device__ int NearestPixel_ROUND_PREFER_CEIL(float x_original, bool) {
-  return static_cast<int>(round(x_original));
```
Comment (edgchen1) [inline onnxruntime/core/providers/cuda/tensor/resize_impl.cu:27]: [round()](https://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH__DOUBLE.html#group__CUDA__MATH__DOUBLE_1gbefba28ee84ef32c44d417cfd4f615d4) has different behavior than [rint()](https://docs.nvidia.com/cuda/cuda-math-api/group__CUDA__MATH__DOUBLE.html#group__CUDA__MATH__DOUBLE_1g3b8026edb2f2e441669845f0f3fa3bf7) (which _Round() uses) for halfway values.
is that ok?

Comment (snnn) [reply onnxruntime/core/providers/cuda/tensor/resize_impl.cu:27]: @duli2012 Any comments?

Comment (edgchen1) [reply onnxruntime/core/providers/cuda/tensor/resize_impl.cu:27]: does it work with roundf()? the argument is a float.

Comment (duli2012) [reply onnxruntime/core/providers/cuda/tensor/resize_impl.cu:27]: round_perfer_ceil means round half up, so round() should be right here?

**Pull request merged**